lib/notifiers/delivery_notifier.dart

```diff
@@ -17,11 +17,17 @@ class DeliveryNotifier extends ChangeNotifier {
     notifyListeners();
   }
 
-  bool deliveryConfirmed() {
-    return _delivery!.state == ZOStrings.deliveryConfirmedState ? true : false;
+  bool deliveryConfirmed(BuildContext context) {
+    return _delivery?.state ==
```

Comment:
to už stačí asi dát result te podmínky, nemusíš znovu dát true/false

Reply:
Fixed